src/index.css

```diff
@@ -162,11 +205,67 @@ card {
   position: fixed;
   width: auto;
   flex-direction: column;
-  background: #c01daa;
+  background-color: #fafafa;
 }
 
 .list-wrapper {
   width: auto;
   height: 20rem;
   overflow-y: scroll;
 }
+
+.head-title {
+  width: auto;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: row;
+  width: auto;
+  height: auto;
+  align-items: center;
+  justify-items: center;
+}
+
+.title-input,
+.author-input {
+  width: 15rem;
+  height: 2rem;
+  margin: 1.1rem;
+  padding: 0.09rem;
+  border-radius: 1px;
+  border: solid 1px var(--white);
+  background-color: #fff;
+}
+
+#category {
+  width: 10rem;
+  height: 2rem;
+}
+
+.input-submit {
+  display: flex;
+  width: 10rem;
+  height: 2.063rem;
+  margin: 1rem;
+  padding: 0;
+  background-color: var(--azure);
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  color: #fff;
```

Comment:
Guideline
![image](https://github.com/emarkees/book-store/assets/78251927/15d6d7fe-f414-4c43-a8d1-ad83b7c76c70)

Your design
![image](https://github.com/emarkees/book-store/assets/78251927/d6138efe-a42b-4aca-8ce1-080a043cbc24)


According to the project requirements please kindly ensure that your design is **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/) 🤗

- Text Color
- Background colors 
- Buttons stylings 👀
- Your final design should match Zeplin's design (this includes all elements + the same styling) 🛠💪

Reply:
@abdulhamiid can i have a sec

Reply:
Hi @emarkees

Reply:
Thank you for your response @abdulhamiid 

my issue is if i remove the scroll from the wrapper the books will will not be scrollable. How can i go about that?

Reply:
But the default behavior of a page should be scrollable. maybe you have `overflow: hidden` somewhere in your CSS which is affecting your design.

Reply:
@abdulhamiid I did hide any overflow. That is the reason I asked if peradventure I did something wrong in my style.
kindly help me review.

Thanks